DATA605/TutorialsTask76_automate_collaborator_invitations_from_gsheet.py

```diff
@@ -0,0 +1,82 @@
+"""
+Import as:
+
+import DATA605.TutorialsTask76_automate_collaborator_invitations_from_gsheet as dtacifrgs
+"""
+
+import logging
+import subprocess
+from typing import List
+
+# Install pygithub.
+subprocess.run(
+    ["sudo", "/venv/bin/pip", "install", "--quiet", " pygithub"], check=True
+)
```

Comment:
Fine for now, but we have a better approach for this which we call "dockerized executables".

You can file an issue to convert this into that approach. You can check with @Shaunak01 about documentation and so on

Reply:
I'll close this to merge. But pls do a follow up.